app/controllers/users_controller.rb

```diff
@@ -114,6 +114,14 @@ def validate_attributes
     if user_params[:role].present? && !current_user.assignable_roles.key?(user_params[:role].to_sym)
       @resource.errors.add :role, I18n.t("validations.role.invalid")
     end
+
+    if user_params[:phone].present? && !valid_phone_number?(user_params[:phone])
```

Comment:
I think it's only here because some of the migrated logs have numbers that otherwise would not pass this validation (for example numbers with extensions etc.)
From ACs:
_If possible - for any newly created numbers (so not those that are imported) include a validation where  all characters are numbers and the number is at least 11 digits long_ 

Let me know if you can think of another way to accomplish this tho